### PR TITLE
Adds known issues for Linux.

### PR DESCRIPTION
**Pull request by kenwith**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Known-issues.md [hunk 1]
```diff
@@ -168,6 +168,44 @@ This article lists the known issues for Microsoft Teams, by feature area.
...
+|:-----|:-----|:-----|:-----|
+|AutoStart on Linux is not working. <br/> |AutoStart on Linux does not start the Teams application. <br/> | <br/> |12/05/19  <br/>|
+
+|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
+|:-----|:-----|:-----|:-----|
+|White screen when resuming from sleep/suspend. <br/> |When your computer resumes / wakes from sleep or suspend mode there can be a network change (especially when the computer is connected to VPN before put to sleep/suspended) and that takes some time for the computer to reobtain the connection. The combination of these things can lead to a Teams white screen. <br/> |Restarting Teams client will help.  <br/> |12/05/19  <br/>|
+
+|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
+|:-----|:-----|:-----|:-----|
+|Cursor missing when screen sharing. <br/> |While sharing screen, other party does not see cursor of person sharing the screen. <br/> | <br/> |12/05/19  <br/>|
```
Comment (JohanFreelancer9) [inline Teams/Known-issues.md]:
```suggestion
|Cursor missing when screen sharing. <br/> |While sharing screen, the other party does not see the cursor of the person sharing the screen. <br/> | <br/> |12/05/19  <br/>|
```

File: Teams/Known-issues.md [hunk 2]
```diff
@@ -168,6 +168,44 @@ This article lists the known issues for Microsoft Teams, by feature area.
...
+|:-----|:-----|:-----|:-----|
+|Package managers not showing changelist. <br/> |Package manager does not show changelist. <br/> | <br/> |12/05/19  <br/>|
+
+|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
+|:-----|:-----|:-----|:-----|
+|Cannot launch Teams client in offline mode. <br/> |Unable to launch Teams Offline in Linux client. <br/> | <br/> |12/05/19  <br/>|
+
+|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
+|:-----|:-----|:-----|:-----|
+|Device settings while in meeting. <br/> |When in a meeting, and changing device settings, the Microphone indicator isn't registering anything being picked up. <br/> | <br/> |12/05/19  <br/>|
```
Comment (JohanFreelancer9) [inline Teams/Known-issues.md]:
```suggestion
|Device settings while in meeting. <br/> |When in a meeting and changing device settings, the Microphone indicator isn't registering anything being picked up. <br/> | <br/> |12/05/19  <br/>|
```

File: Teams/Known-issues.md [hunk 3]
```diff
@@ -168,6 +168,44 @@ This article lists the known issues for Microsoft Teams, by feature area.
...
+
+|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
+|:-----|:-----|:-----|:-----|
+|Can't close Teams application using keyboard. <br/> |Can't close using the default `$mod + shift + q` or by clicking the close button in the app. <br/> | <br/> |12/05/19  <br/>|
```
Comment (JohanFreelancer9) [inline Teams/Known-issues.md]:
```suggestion
|Can't close Teams application using keyboard. <br/> |Can't close the Teams application using the default `$mod + shift + q` or by clicking the close button in the app. <br/> | <br/> |12/05/19  <br/>|
```

**Pull request merged**